datalad_get_single_dataset.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+#step1
+ds_id="ds004107"
+
+ds_portal="https://openneuro.org/datasets/${ds_id}"
+GIT="git@github.com:OpenNeuroDatasets-JSONLD/${ds_id}"
+bagel_dir="/home/maellef/DataBase/NeuroBagel_test/"
+ldout="${bagel_dir}outputs/openneuro-jsonld/"
```

Comment:
It may make sense to keep `bagel_dir` and `ldout` as relative directories for now instead of hardcoding the absolute file paths, to avoid problems when we launch this from inside a Docker container (which has its own file system).

Maybe something like this?
```bash
bagel_dir="inputs/openneuro-jsonld/"
ldout="outputs/openneuro-jsonld/"
```